### PR TITLE
Add Systemd integration to aes67-daemon

### PR DESCRIPTION
**Pull request by glance-**
This adds integration and especially watchdog support between aes67-daemon and systemd.

A example service file is also included.

### CODE REVIEW

File: daemon/main.cpp [hunk 1]
```diff
@@ -31,6 +31,13 @@
 #include "rtsp_server.hpp"
 #include "session_manager.hpp"
 
+#ifdef _USE_SYSTEMD_
+#include <systemd/sd-daemon.h>
+
+// with which interval we should pet the dog
+uint64_t current_watchdog_usec;
```
Comment (bondagit) [inline daemon/main.cpp:38]: move this declaration into main() function.

**Comment by bondagit** [comment]
Please request a merge into [daemon_v1.6.1](https://github.com/bondagit/aes67-linux-daemon/tree/daemon_v1.6.1)
See comments and indent your code using clang-format.

File: daemon/main.cpp (hunk 1 above)
Comment (glance-) [reply daemon/main.cpp:38]: Was mainly lazy, and re-used the ifdef. I'll move it.

Comment (glance-) [reply daemon/main.cpp]: Now it's fixed.

**Pull request merged**